src/main/java/com/example/resourcemanagerapp/controller/MyResourceController.java

```diff
@@ -0,0 +1,85 @@
+package com.example.resourcemanagerapp.controller;
+
+
+import com.example.resourcemanagerapp.additionalTypes.EnumChecker;
+import com.example.resourcemanagerapp.additionalTypes.MyResourceType;
+import com.example.resourcemanagerapp.service.MyResourceServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.configurationprocessor.json.JSONException;
+import org.springframework.boot.configurationprocessor.json.JSONObject;
```

Comment:
Generalnie za duzo logiki w controllerach, tak naprawde jedyna logika jaka powinna tu byc to wywolanie serwisy lub ewentualne operacje na http request/response. Przenies logike biznesowa do serwisu